samples/editors/date-picker/overview/App.razor

```diff
@@ -3,14 +3,17 @@
 
 <div class="container vertical">
     <div class="container vertical">
-        <DatePicker Height="50px" Width="220px" Value="@SelectedDate" />
+        <DatePicker @ref="DatePicker" Height="50px" Width="220px" 
```

Comment:
increase height to 60px such that input box has same height as a date in popup box:
![image](https://user-images.githubusercontent.com/11665978/114919769-a9219c80-9df6-11eb-9c0f-db134ac60a1f.png)

Reply:
pending